crates/release-automation/src/lib/crate_selection/mod.rs

```diff
@@ -99,7 +99,8 @@ impl<'a> Crate<'a> {
         release_version: &semver::Version,
     ) -> Fallible<Vec<&'a Crate<'a>>> {
         debug!(
-            "setting version to {} in manifest at {:?}",
+            "[{}] setting version to {} in manifest at {:?}",
+            self.name(),
```

Comment:
@steveej Are you fine with outputting the package name in the debug logs?

Reply:
yes and it's already over here :-) https://github.com/holochain/holochain/pull/2896#discussion_r1346350597

Reply:
this is not a duplicate as i thought it'd be. change is fine with me